integrationtest-service-framework/src/main/java/org/hypertrace/core/serviceframework/config/IntegrationTestConfigClient.java

```diff
@@ -14,11 +14,19 @@ public class IntegrationTestConfigClient implements ConfigClient {
   private static final String CONFIGS_PREFIX = "configs/";
   private static final String INTEGRATION_TEST_COMMON_DIRECTORY = "common";
   private static final String INTEGRATION_TEST_CLUSTER = "local";
+  private static final String DEFAULT_TEST_NAME = "test";
 
   private final String defaultServiceName;
+  private final String defaultTestName;
 
   public IntegrationTestConfigClient(String defaultServiceName) {
     this.defaultServiceName = defaultServiceName;
+    this.defaultTestName = DEFAULT_TEST_NAME;
```

Comment:
do we need a default test name? When would this be used, since we already have a config scoped to all tests and the functionality being added is for a specific test.

Reply:
removed it